data/PlantGrowthFormCurated_2022/metadata.yml

```diff
@@ -0,0 +1,70 @@
+source:
+  primary:
+    key: PlantGrowthFormCurated_2022
+    bibtype: Article
+    year: unknown
+    author: unknown
+    title: unknown
+    journal: unknown
+    volume: unknown
+    number: unknown
+    pages: unknown
+    doi: unknown
+contributors:
+  data_collectors:
+  - last_name: unknown
+    given_name: unknown
+    ORCID: unknown
+    affiliation: unknown
+    additional_role: unknown
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no  
+  custom_R_code:  '
+    data %>%
+      filter(str_detect(taxon_name, " ")) %>%
+      mutate(
+        plant_growth_form_to_use = stringr::str_replace(
+          plant_growth_form_to_use, "monocot_shrub", "basal_large"),                
+        plant_growth_form_to_use = stringr::str_replace(
+          plant_growth_form_to_use, "monocot_large_basal", "basal_large"),
+        woodiness_to_use = stringr::str_replace(
+          woodiness_to_use, "soft_woody", "semi_woody"),
+        woodiness_to_use = stringr::str_replace(
+          woodiness_to_use, "woody_stemmed_monocot", "woody_like_stem"),
+      )
+  '
+  collection_date: unknown/2022
+  taxon_name: taxon_name
+  description: A curated collection of plant growth form values that includes all species in the Australian Plant Census (APC).
```

Comment:
What's the basis for including the curated dataset as a standalone dataset? ARE duplicates with all other floras? I think it's answered in sampling strategy, but suggest making clear what's included here and why. 

For example, is this dataset all plants in APC (as it says), or just those not in the other sources?

Reply:
There are duplicates, but the `curated` dataset represents a consensus across floras with a significant number of taxa (4000+) being filled in manually and many more where the description was read to confirm the automated score.